src/main/java/com/stuypulse/robot/commands/ClimberMoveYoyoCommand.java

```diff
@@ -0,0 +1,31 @@
+package com.stuypulse.robot.commands;
+
+import com.stuypulse.robot.Constants;
+import com.stuypulse.robot.subsystems.Climber;
+import com.stuypulse.stuylib.input.Gamepad;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ClimberMoveYoyoCommand extends CommandBase{
+
+    Climber climber;
+    Gamepad gamepad;
+
+    public ClimberMoveYoyoCommand(Climber climber, Gamepad gamepad) {
+        this.climber = climber;
+        this.gamepad = gamepad;
+        
+        addRequirements(climber);
+    }
+
+    @Override
+    public void execute() {
+        climber.moveYoyo(Math.cbrt(gamepad.getLeftX() * Constants.CLIMBER_SCALE));
```

Comment:
This should be done using the Math.pow() command so that we can easily change the power it is raised to